docs/tutorials/standalone/Bucket/coupled_bucket.jl

```diff
@@ -74,31 +74,33 @@
 # struct CoupledRadiativeFluxes{FT} <: AbstractRadiativeDrivers{FT} end
 # ```
 
-# Then, we have defined a new method for `turbulent_fluxes` and `net_radiation` which dispatch for these types,
+# Then, we have defined a new method for `turbulent_fluxes!` and `net_radiation!` which dispatch for these types,
 # and simply return the fluxes that the coupler has updated `p.bucket.turbulent_fluxes` and `p.bucket.R_n` with.
```

Comment:
"and simply update `dest` in place with the fluxes that the coupler has updated `p.bucket.turbulent_fluxes` and `p.bucket.R_n` with."

Reading this, I actually think this function can do nothing in this case. Because we are setting p.bucket.Rn .= p.bucket.Rn, basically!

We can leave alone for now and do quick followup PR or you can change it? up to you